src/Docnet.Core/Validation/IPageRangeValidator.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Docnet.Core.Validation
+{
+    public interface IPageRangeValidator
```

Comment:
why do we need this interface? we do not ever expect to mock it or inject a different page range validation behavior anywhere so this just introduces unnecessary indirection.

Reply:
Removed